docs(api): lids updates in 8.4 (#17914)
OpenTrons/opentrons-api: api/docs/v2/moving_labware.rst

          * ``opentrons_flex_96_filtertiprack_50ul``
          * ``opentrons_flex_96_filtertiprack_200ul``
          * ``opentrons_flex_96_filtertiprack_1000ul``
    * - Opentrons labware lids 
      - 
          * ``opentrons_tough_pcr_auto_sealing_lid``
          * ``opentrons_flex_tiprack_lid``
    

> You can move compatible Opentrons lids manually or with the Flex gripper, but some restrictions apply. For more, see :ref:`moving-lids`. 

Reviewer: This is minor, the "For more" perhaps should be:

```suggestion
You can move compatible Opentrons lids manually or with the Flex gripper, but some restrictions apply. For more information, see :ref:`moving-lids`. 
```

or just "See `moving-lids`"

            pipette.pick_up_tip()

Using the off-deck location to remove or replace labware lets you continue your workflow in a single protocol, rather than needing to end a protocol, reset the deck, and start a new protocol run.

.. _moving-lids:

Moving Lids 
===========

Lids on well plates or tip racks can help prevent contamination on the deck and are required for use with some modules, like the Thermocycler. You can use :py:meth:`.ProtocolContext.move_lid` to move an ``opentrons_tough_pcr_auto_sealing_lid`` or ``opentrons_flex_tiprack_lid`` manually or using the Flex gripper. 

> An `opentrons_tough_pcr_auto_sealing_lid` can be moved between deck slots, lid stacks, or compatible labware, modules, and adapters loaded in your protocol. 

Reviewer: Sometimes, a bit of text can read easer if the noun is referred to by name rather than by code. If you're explicitly referring to a code example, then I'd use the `code` typeface, but this seems more like a "general" type example so I'd suggest regular text because the sentence is about the actual object, not the code.

```suggestion
An Opentrons Tough PCR lid can be moved between deck slots, lid stacks, or compatible labware, modules, and adapters loaded in your protocol. 
```

Reviewer 2: I agree in general, prefer plain English names unless directly referring to something in a code block.

.. code-block:: python

    ## load lid stack on the Flex deck 

    lid_stack = protocol.load_lid_stack(
        load_name="opentrons_tough_pcr_auto_sealing_lid",
        location="B2", 
        quantity="4")
    
    ## move an Opentrons Tough PCR Auto-Sealing Lid to a compatible well plate in the Thermocycler

>     protocol.move_lid(

Author: @CaseyBatten do we not specify the load name for the lid being moved?

        source_location=("lid_stack", "B2"),
        new_location=("opentrons_96_wellplate_200ul_pcr_full_skirt", "tcmod"),
        use_gripper= "True")

Use ``move_lid()`` again to move a used ``opentrons_tough_pcr_auto_sealing_lid`` to a ``WasteChute`` or ``TrashBin`` loaded in your protocol. 

.. note::

>     You can only move an ``opentrons_flex_tip_rack_lid`` from a new tip rack to the ``WasteChute`` or ``TrashBin``, either manually or with the Flex gripper. The tip rack lid must be defined in your protocol using the ``lid`` parameter of ``load_labware``. For more, see :ref:`loading-lids`.

Reviewer 3: You can also move them onto another tip rack which does not currently have a lid. The example case being, you load one tiprack with a lid and one tiprack without a lid. On step two, you move the lid from tiprack A to tiprack B.

> Use ``move_lid()`` again to move a used ``opentrons_tough_pcr_auto_sealing_lid`` to a ``WasteChute`` or ``TrashBin`` loaded in your protocol. 
> 
> .. note::
>     You can only move an ``opentrons_flex_tip_rack_lid`` from a new tip rack to the ``WasteChute`` or ``TrashBin``, either manually or with the Flex gripper. The tip rack lid must be defined in your protocol using the ``lid`` parameter of ``load_labware``. For more, see :ref:`loading-lids`.

Reviewer: I would suggest the same thing as my comment above, though this might be an ambiguous case. Consider this:

```suggestion
Use ``move_lid()`` again to move a used Opentrons Tough PCR lid to a waste chute or trash bin loaded in your protocol. 

.. note::
    You can only move an Opentrons Tough PCR lid from a new tip rack to the  waste chute or trans bin, either manually or with the Flex gripper. The tip rack lid must be defined in your protocol using the ``lid`` parameter of ``load_labware``. For more information, see :ref:`loading-lids`.
```

The "note" is a good example here. You're moving a lid not a piece of code to other real, physical objects.

Now the last sentence of the note is ok because it is talking about code.
